code/game/objects/items/tools/wrench.dm

```diff
@@ -41,7 +41,7 @@
 	force = 2 //MEDICAL
 	throwforce = 4
 
-	attack_verb = list("wrenched", "medicaled", "tapped", "jabbed", "whacked")
+	attack_verb = list("healed", "medicaled", "tapped", "poked", "analyzed") //"cobbyed"

```

Comment:
`//"cobbyed"`

Nice

Reply:
if the cobbyed message plays it deals 10 liver damage